lms/djangoapps/certificates/management/commands/fix_cert_records.py

```diff
@@ -0,0 +1,54 @@
+"""
+(Temporary) Management command used to fix some incorrectly generated certificate records created as a side effect of
+CR-3792.
+"""
+import datetime
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from common.djangoapps.student.models import CourseEnrollment
+from lms.djangoapps.certificates.generation_handler import _generate_certificate_task
+from lms.djangoapps.certificates.models import GeneratedCertificate
+
+log = logging.getLogger(__name__)
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """
+    Temporary Management command to fix the incorrect mode on a number of records in the
+    `CERTIFICATES_GENERATEDCERTIFICATE table.
+    """
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-l', '--limit',
+            metavar='LIMIT',
+            dest='limit',
+            help='number of records to process at once'
+        )
```

Comment:
This management command will spawn a Celery task for each record in the `CERTIFICATES_GENERATEDCERTIFICATE` table we need to update. Currently, at the time of writing this comment, we are looking at creating roughly 23,500 tasks.

I wanted to try and control how many tasks we are creating at once. I tried to follow the `Paginator` examples we used in recent data migrations but kept running into issues unit testing my changes where records weren't being updated. I was pulling my hair out trying to figure it out.

I am expecting to be throwing this code away so instead of spending more time trying to troubleshoot the issues I decided to move towards passing in a `limit` value. This requires us to run the management command a few times, but we will be able to control how many records we process at once.